This function was stabilized in rust 1.82

error[E0599]: no method named `is_none_or` found for enum `Option` in the current scope
  --> ../../third_party/rust/chromium_crates_io/vendor/skrifa-0.29.2/src/glyph_name.rs:97:26
   |
97 |         if name.as_ref().is_none_or(|s| s.is_empty()) {
   |                          ^^^^^^^^^^
   |
help: there is a method `is_none` with a similar name, but with different arguments
  --> /usr/src/rustc-1.78.0/library/core/src/option.rs:651:5

--- a/third_party/rust/chromium_crates_io/vendor/skrifa-v0_29/src/glyph_name.rs
+++ b/third_party/rust/chromium_crates_io/vendor/skrifa-v0_29/src/glyph_name.rs
@@ -94,7 +94,7 @@ impl<'a> GlyphNames<'a> {
             _ => None,
         };
         // If name is empty string, synthesize it
-        if name.as_ref().is_none_or(|s| s.is_empty()) {
+        if name.as_ref().is_none() || name.as_ref().is_some_and(|s| s.is_empty()) {
             return Some(GlyphName::synthesize(glyph_id));
         }
         Some(name.unwrap_or_else(|| GlyphName::synthesize(glyph_id)))
--- a/third_party/rust/chromium_crates_io/vendor/llguidance-v0_7/src/ffi.rs
+++ b/third_party/rust/chromium_crates_io/vendor/llguidance-v0_7/src/ffi.rs
@@ -556,7 +556,7 @@ pub extern "C" fn llg_get_temperature(cc
 pub extern "C" fn llg_is_stopped(cc: &LlgConstraint) -> bool {
     cc.constraint
         .as_ref()
-        .is_none_or(|c| c.step_result().is_stop())
+        .is_none() || cc.constraint.as_ref().is_some_and(|c| c.step_result().is_stop())
 }
 
 /// Compute mask for the next token sampling
